src/main/java/seedu/address/logic/commands/wallet/WalletFindCommand.java

```diff
@@ -21,9 +21,11 @@
             + "Example: " + WALLET_COMMAND_TYPE + " "
             + COMMAND_WORD + " rice water chicken";
 
-    private final DescriptionContainsKeywordsPredicate predicate;
+    public static final String NO_PARAMETER_INPUTTED = "At least one field to find must be provided.";
```

Comment:
for people find command i just 
`throw new ParseException(
                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, PeopleFindCommand.MESSAGE_USAGE));`
in the PeopleFindCommandParser leh...

Reply:
True I think they did it for original also